src/tic_tac_toe/referee.clj

```diff
@@ -8,8 +8,14 @@
   [move]
   (- move 1))
 
-(defn validate-move
-  [move]
+(defmulti validate-move (fn [player move] (:type player)))
```

Comment:
Be careful of multimethods. In a way, this is similar to this kind of code

```ruby
def validate-move(player, move)
  if player.instanceof? Human
     # validate human move
  else if player.instanceof? Computer
     # validate computer move
  else
     # something else
end
```
This would be ok if we did it once maybe, but this is the second multimethod that is dispatching on `:human` and `:computer`. Is there a way we could do this more polymorphically?

Reply:
Changed this to use [functions](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-f014d0e7db79dee6fc42af72f74d5325R5) on ref which can accommodate both human and computer inputs.